ddp-workspace/projects/study-builder-ui/src/app/components/validators-block/validators-block.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { BehaviorSubject, merge, Observable, Subscription } from 'rxjs';
+import { filter, tap } from 'rxjs/operators';
+import * as _ from 'underscore';
+import { ValidationControlsData, ValidatorsMapper } from './validators.mapper';
+import { RuleDef } from '../../model/core/ruleDef';
+
+@Component({
+    selector: 'app-validators-block',
+    templateUrl: './validators-block.component.html',
+    styleUrls: ['./validators-block.component.scss']
+})
+export class ValidatorsBlockComponent implements OnInit, OnDestroy {
+    @Input() set validators(data: RuleDef[]) {
+        if (data) {
+            this.validatorsDataSubject.next(ValidatorsMapper.mapToValidationControlsData(data));
+        }
+    }
+    @Output() validatorsChanged = new EventEmitter<any>();
+
+    validatorsDataSubject = new BehaviorSubject<ValidationControlsData | boolean>(false);
```

Comment:
Use boolean values for show/hide validators block (vs "Add validators" button).
![image](https://user-images.githubusercontent.com/7396837/147824944-7911639a-673d-4449-9ecd-84dda6a7143b.png)

Reply:
Made a comment somewhere else (duplicated) but have preference for showing the list of available validators here/without having to first click on button.
You already implemented enabling/disabling individual validators and I think that is sufficient.

Reply:
Done.